sdk/keyvault/azkeys/client.go

```diff
@@ -565,7 +569,7 @@ func purgeDeletedKeyResponseFromGenerated(i generated.KeyVaultClientPurgeDeleted
 
 // PurgeDeletedKey deletes the specified key. The purge deleted key operation removes the key permanently, without the possibility of recovery.
 // This operation can only be enabled on a soft-delete enabled vault. This operation requires the key/purge permission.
-// Pass nil to use the default options.
+// Pass nil for options to accept default values.
```

Comment:
Given that this doesn't really complete after the API returns, seems like a comment should be added to indicate this quirk.